<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QListView" name="Cut1D_listView">
       <property name="toolTip">
        <string>List of generated cuts</string>
       </property>
       <property name="statusTip">
        <string>List of generated cuts</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="1" column="1">
          <widget class="QLineEdit" name="Cut1D_HStart_lineEdit">
           <property name="toolTip">
            <string>Start position for cut</string>
           </property>
           <property name="statusTip">
            <string>Start position for cut</string>
           </property>
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="Cut1D_Llabel">
           <property name="toolTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="statusTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="text">
            <string>L</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Cut1D_StartLabel">
           <property name="toolTip">
            <string>Start position for cut</string>
           </property>
           <property name="statusTip">
            <string>Start position for cut</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="Cut1D_KEnd_lineEdit">
           <property name="toolTip">
            <string>End position for cut</string>
           </property>
           <property name="statusTip">
            <string>End position for cut</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Cut1D_HEnd_lineEdit">
           <property name="toolTip">
            <string>End position for cut</string>
           </property>
           <property name="statusTip">
            <string>End position for cut</string>
           </property>
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="Cut1D_LEnd_lineEdit">
           <property name="toolTip">
            <string>End position for cut</string>
           </property>
           <property name="statusTip">
            <string>End position for cut</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="Cut1D_LStart_lineEdit">
           <property name="toolTip">
            <string>Start position for cut</string>
           </property>
           <property name="statusTip">
            <string>Start position for cut</string>
           </property>
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Cut1D_StopLabel">
           <property name="toolTip">
            <string>End position for cut</string>
           </property>
           <property name="statusTip">
            <string>End position for cut</string>
           </property>
           <property name="text">
            <string>End</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="Cut1D_KStart_lineEdit">
           <property name="toolTip">
            <string>Start position for cut</string>
           </property>
           <property name="statusTip">
            <string>Start position for cut</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="Cut1D_Hlabel">
           <property name="toolTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="statusTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="text">
            <string>H</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="Cut1D_Klabel">
           <property name="toolTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="statusTip">
            <string>Unit of 1D cut</string>
           </property>
           <property name="text">
            <string>K</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Energy [meV]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>Minimum energy in meV</string>
           </property>
           <property name="statusTip">
            <string>Minimum energy in meV</string>
           </property>
           <property name="text">
            <string>Min</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="toolTip">
            <string>Width orthogonal to cut over which integration is performed</string>
           </property>
           <property name="statusTip">
            <string>Width orthogonal to cut over which integration is performed</string>
           </property>
           <property name="text">
            <string>Q width [1/AA]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="Cut1D_EMax_lineEdit">
           <property name="toolTip">
            <string>Maximum energy in meV</string>
           </property>
           <property name="statusTip">
            <string>Maximum energy in meV</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="Cut1D_SelectCut_E_radioButton">
           <property name="toolTip">
            <string>Perform cut along E for constant Q</string>
           </property>
           <property name="statusTip">
            <string>Perform cut along E for constant Q</string>
           </property>
           <property name="text">
            <string>Cut along E</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="Cut1D_MinPixel_lineEdit">
           <property name="toolTip">
            <string>Integration step size along cut</string>
           </property>
           <property name="statusTip">
            <string>Integration step size along cut</string>
           </property>
           <property name="text">
            <string>0.03</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="Cut1D_EMin_lineEdit">
           <property name="toolTip">
            <string>Minimum energy in meV</string>
           </property>
           <property name="statusTip">
            <string>Minimum energy in meV</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="Cut1D_SetTitle_lineEdit">
             <property name="toolTip">
              <string>Set title of current figure</string>
             </property>
             <property name="statusTip">
              <string>Set title of current figure</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Cut1D_SetTitle_button">
             <property name="toolTip">
              <string>Set title of current figure</string>
             </property>
             <property name="statusTip">
              <string>Set title of current figure</string>
             </property>
             <property name="text">
              <string>Set Title</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="Cut1D_SelectCut_Q_radioButton">
           <property name="toolTip">
            <string>Perform cut along Q for constant E</string>
           </property>
           <property name="statusTip">
            <string>Perform cut along Q for constant E</string>
           </property>
           <property name="text">
            <string>Cut along Q</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup_2</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="Cut1D_MinPixel_label">
           <property name="toolTip">
            <string>Integration step size along cut</string>
           </property>
           <property name="statusTip">
            <string>Integration step size along cut</string>
           </property>
           <property name="text">
            <string>Min Pixel [1/AA]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="Cut1D_Width_lineEdit">
           <property name="toolTip">
            <string>Width orthogonal to cut over which integration is performed</string>
           </property>
           <property name="statusTip">
            <string>Width orthogonal to cut over which integration is performed</string>
           </property>
           <property name="text">
            <string>0.2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>Maximum energy in meV</string>
           </property>
           <property name="statusTip">
            <string>Maximum energy in meV</string>
           </property>
           <property name="text">
            <string>Max</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>Units used for cut</string>
             </property>
             <property name="statusTip">
              <string>Units used for cut</string>
             </property>
             <property name="text">
              <string>Units</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Cut1D_SelectUnits_RLU_radioButton">
             <property name="toolTip">
              <string>Units used for cut</string>
             </property>
             <property name="statusTip">
              <string>Units used for cut</string>
             </property>
             <property name="text">
              <string>RLU</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Cut1D_SelectUnits_AA_radioButton">
             <property name="toolTip">
              <string>Units used for cut</string>
             </property>
             <property name="statusTip">
              <string>Units used for cut</string>
             </property>
             <property name="text">
              <string>AA-1</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1" columnstretch="0,0">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="Cut1D_plot_button">
           <property name="toolTip">
            <string>Generate and plot 1D cut and add to list</string>
           </property>
           <property name="statusTip">
            <string>Generate and plot 1D cut and add to list</string>
           </property>
           <property name="text">
            <string>Plot 1D cut</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="Cut1D_Generate1D_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Generate 1D cut and add to list</string>
           </property>
           <property name="statusTip">
            <string>Generate 1D cut and add to list</string>
           </property>
           <property name="text">
            <string>Generate 1D cut</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="Cut1D_Delete1D_btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Delete current cut(s) from list</string>
           </property>
           <property name="statusTip">
            <string>Delete current cut(s) from list</string>
           </property>
           <property name="text">
            <string>Delete 1D cut</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="Cut1D_plotTogether_btn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Plot Selected 1D Cuts together</string>
           </property>
           <property name="statusTip">
            <string>Plot Selected 1D Cuts together</string>
           </property>
           <property name="text">
            <string>Plot Cuts together</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="toolTip">
      <string>Export current list of cuts</string>
     </property>
     <property name="statusTip">
      <string>Export current list of cuts</string>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QRadioButton" name="Cut1D_ExporCSV_radioBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>csv</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Cut1D_ExporUFIT_radioBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>uFit</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Cut1D_ExportName_lineEdit">
        <property name="toolTip">
         <string>Name of exported file(s) - Include a '#' to insert given name of cut</string>
        </property>
        <property name="statusTip">
         <string>Name of exported file(s) - Include a '#' to insert given name of cut</string>
        </property>
        <property name="text">
         <string>1DCut_#.csv</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cut1D_Export1D_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export All</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Cut1D_Generate1D_button</tabstop>
  <tabstop>Cut1D_plot_button</tabstop>
  <tabstop>Cut1D_Delete1D_btn</tabstop>
  <tabstop>Cut1D_HStart_lineEdit</tabstop>
  <tabstop>Cut1D_KStart_lineEdit</tabstop>
  <tabstop>Cut1D_LStart_lineEdit</tabstop>
  <tabstop>Cut1D_HEnd_lineEdit</tabstop>
  <tabstop>Cut1D_KEnd_lineEdit</tabstop>
  <tabstop>Cut1D_LEnd_lineEdit</tabstop>
  <tabstop>Cut1D_EMin_lineEdit</tabstop>
  <tabstop>Cut1D_EMax_lineEdit</tabstop>
  <tabstop>Cut1D_Width_lineEdit</tabstop>
  <tabstop>Cut1D_MinPixel_lineEdit</tabstop>
  <tabstop>Cut1D_SelectCut_Q_radioButton</tabstop>
  <tabstop>Cut1D_SelectCut_E_radioButton</tabstop>
  <tabstop>Cut1D_SetTitle_lineEdit</tabstop>
  <tabstop>Cut1D_SetTitle_button</tabstop>
  <tabstop>Cut1D_SelectUnits_RLU_radioButton</tabstop>
  <tabstop>Cut1D_SelectUnits_AA_radioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
